net-im/synapse/synapse-1.74.0.ebuild

```diff
@@ -127,6 +127,7 @@ RDEPEND="${DEPEND}
 	systemd? ( dev-python/python-systemd[${PYTHON_USEDEP}] )
```

Comment:
This needs a new revision as it's "miscompiled" (misinstalled, really) with newer versions.

Reply:
Oh, yeah, I see. I have revbumped 1.74.0 and 1.75.0. I left 1.76.0 as it was because it is not in the tree yet.